logger_121.go

```diff
@@ -27,7 +36,7 @@ func (l *Logger) Handle(ctx context.Context, record slog.Record) error {
 
 	fields := make([]interface{}, 0, record.NumAttrs()*2)
 	record.Attrs(func(a slog.Attr) bool {
-		fields = append(fields, a.Key, a.Value.String())
+		fields = append(fields, a.Key, a.Value)
```

Comment:
Probably want to do the same for `logger_no121.go`

Reply:
Fixed!